### PR TITLE
Updated: Added links for Online WWCode Greenville presence.

### PR DESCRIPTION
**Pull request by prakartigoel24**
Please add "hacktoberfest" and "hacktoberfest-accepted" Label to my PR.
This PR fixes issue #59.

 - Added a Find us online section to show all the links .
 - Added links for Meetup
 - Facebook
 - X (Twitter)
 - WWCode Greenville Website
 - Open Collective website 

# Check image here : 
 

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +97,19 @@ WWCodeGreenville - [@wwcodegville](https://twitter.com/wwcodegville) - greenvill
 
 Project Link: [https://github.com/WomenWhoCode/WWCodeGreenville](https://github.com/WomenWhoCode/WWCodeGreenville)
 
+<!-- CONTACT -->
+## Find us online
```
Comment (JenBauer) [inline README.md:101]: Can this be changed to "Find Us Online" (proper case)?  Thanks!

Comment (JenBauer) [reply README.md:101]: Also, can two more sites be added in here: the Instagram account and Notion account links from this Link Tree page? https://linktr.ee/wwcodegville

**Review by JenBauer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (prakartigoel24) [reply README.md:101]: Sure, making these changes asap.

**Comment by prakartigoel24** [comment]
@JenBauer I have updated the Readme.md with the proposed changes.
Also, please add 'hacktoberfest-accepted' label to my PR. Thank you so much!

**Review by JenBauer — APPROVED** [review]
Thank you for the updates.  Approved.

**Pull request merged**